dev/build_release.py

```diff
@@ -594,3 +595,14 @@ def do_build(cls, options, build_number, container_builder):
 
       print '\nFINISHED writing release to {rep}'.format(
         rep=options.bintray_repo)
+
+  @classmethod
+  def main(cls):
+    parser = argparse.ArgumentParser()
+    cls.init_argument_parser(parser)
+    options = parser.parse_args()
+    # builds debians only
+    cls.do_build(options, '', None)
```

Comment:
We removed the `--build_number` flag from `Builder` (if it ever existed there) because of name collisions in https://github.com/spinnaker/spinnaker/blob/master/dev/build_prevalidation.py#L35. You can set it with an environment variable: https://github.com/spinnaker/spinnaker/blob/master/dev/build_release.py#L126